custom_components/ai_thermostat/climate.py

```diff
@@ -209,6 +210,8 @@ def __init__(
 		self._today_nightmode_end = datetime.now()
 		self.local_temperature_calibration_entity = None
 		self.valve_position_entity = None
+		self.last_change = HVAC_MODE_OFF
```

Comment:
Please use `None` and catch this default value in your code to avoid unforeseeable bugs where this initially set value is processed as valid data